Dockerfile

```diff
@@ -1,10 +1,10 @@
 # WARNING: We must pin the Alpine version to 3.8 because freeradius-rest (r6) package in alpine-3.9 contains
```

Comment:
We can remove the comment as it no longer applies, I wouldn't want it to confuse people in the future.

Reply:
Done